Analytical_Engine/Query/Geometry.cs

```diff
@@ -171,5 +171,17 @@ public static Face Geometry(this IFace face)
             return geomFace;
         }
 
+        /***************************************************/
+
+        [Description("Gets the geometry of a IRegion as its Perimiter curve. Method required for automatic display in UI packages.")]
```

Comment:
Not only for that, but also for `RunExtensionMethod` pattern - may be worth mentioning

Reply:
hmm, true. Not sure if that adds info to the user or not though? Saying that, not sure the current comment does either!

Reply:
Not sure about the user, but could be good to capture that in the API? I am not sure how many people will reach the `RunExtensionMethod` Wiki page.

The above may be a more general question to all get/set geometry methods, therefore happy to approve the PR as is.